mentat/session.py

```diff
@@ -53,13 +53,39 @@ def __init__(
         sentry_init()
         self.id = uuid4()
 
+        try:
+            # This is so that if we set the session context in a test, we don't reset it;
+            # is there a better way to do this?
```

Comment:
@waydegg

Reply:
It's not that I want the session context to persist between different tests; it's that we mock the session context in a fixture, and I don't want Session overwriting the mocked sessioncontext. Thinking about it more, I think the best thing to do would actually be to create a Session in a fixture before we mock sessioncontext and then to use that Session fixture in our tests.